neptune/alpha/internal/operation_processors/sync_operation_processor.py

```diff
@@ -29,8 +29,9 @@ def __init__(self, experiment_uuid: uuid.UUID, backend: NeptuneBackend):
 
     def enqueue_operation(self, op: Operation, wait: bool) -> None:
         # pylint: disable=unused-argument
-        # TODO: Handle metadata errors
-        self._backend.execute_operations(self._experiment_uuid, [op])
+        errors = self._backend.execute_operations(self._experiment_uuid, [op])
+        if errors:
+            raise errors[0]
```

Comment:
How errors are handled in async_operation_processor ?

Reply:
Printed to stderr.